### PR TITLE
VZ-7006 Additional HA configurations

### PR DESCRIPTION
**Pull request by stevshap**
This PR adds the following configuration YAML's:
`ha-oci-ccm.yaml` Using OCNE
`ha-ext-lb.yaml` Using external LB's

### CODE REVIEW

**Review by pfmackin — APPROVED** [review]

File: examples/ha/README.md [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 The `ha.yaml` file is provided as an example of how to achieve high availability using the `prod` profile.
 
-For more information on how to use the `ha.yaml` file, see [Configure High Availability](https://build.verrazzano.io/latest/docs/setup/customizing/ha/).
+The `ha-oci-ccm.yaml` file is provided as an example of how to achieve high availability using the `prod` profile on OCNE.
+
+The `ha-ext-lb.yaml` file is provided as an example of how to achieve high availability using the `prod` profile and external load balancers.
+
+For more information on how to use the `ha.yaml` files, see [Configure High Availability](https://build.verrazzano.io/latest/docs/setup/customizing/ha/).
```
Comment (mgianatagh) [inline examples/ha/README.md:9]: There are now three different file names.  Maybe reword to something like "For more information on how to use these example files,"

Comment (stevshap) [reply examples/ha/README.md:9]: ok
